src/main/java/com/additional/library/domain/Book.java

```diff
@@ -14,13 +14,13 @@
 @Setter
 public class Book {
 
-    private int id;
+    private Long id;
     private String title;
     private String author;
-    private int publicationYear;
+    private Long publicationYear;
```

Comment:
a po co ta zmiana? :) dla roku to chyba wystarczy int

Reply:
Ahh, za bardzo się rozpędziłem :)